container-images/mds-compliance-service/webpack.config.ts

```diff
@@ -0,0 +1,3 @@
+import webpack from '@mds-core/mds-webpack-config'
+
+export default webpack.Bundle('../../packages/mds-jurisdiction-service/server').UsingDefaultConfig()
```

Comment:
Wrong service, should have a path to `../../packages/mds-compliance-service/server`

Reply:
opes